yarn.lock

```diff
@@ -1875,40 +2185,51 @@
     "@types/bs58" "^4.0.0"
     bs58 "^4.0.1"
 
-"@polkadot/rpc-core@^0.46.9":
-  version "0.46.9"
-  resolved "https://registry.yarnpkg.com/@polkadot/rpc-core/-/rpc-core-0.46.9.tgz#fd46312ed5a9da5d1bdefbbd957f641686fec215"
-  integrity sha512-mBMpt/G5NGQal9xPnqRrlTc87Hn4GHsqJzePHwKNBRIEpzXnQfthIKUgFhhAhrmItROkDYrw8zwLD+DVO/sA2A==
+"@polkadot/keyring@^0.36.3":
```

Comment:
Could you run also `rm yarn.lock && yarn install` once? I see 3 different versions of `@polkadot/keyring` here.